server.js

```diff
@@ -1,37 +1,57 @@
-const express = require('express')
+const express = require("express");
+const http = require("http");
+const app = express();
+const server = http.createServer(app);
+const socket = require("socket.io");
 const cors = require('cors')
-
-const app = express()
-
-app.use(cors())
-
-app.get('/', (req, res) => {
-  res.send('Server is running')
-})
-
-const server = app.listen(process.env.PORT || 8080)
-const io = require('socket.io')(server, {
+const io = socket(server, {
   cors: {
     origin: '*',
     methods: ['GET', 'POST'],
   },
-})
-io.on('connection', (socket) => {
-  console.log('connection!', socket.id)
-  socket.on('getId', () => {
-    socket.emit('me', socket.id)
-  })
+});
+
+const users = {};
+
+const socketToRoom = {};
```

Comment:
We probably want to use a database for permanent file storage instead of this. At least... in production.